migrations/migration-9/pin-set.js

```diff
@@ -164,14 +160,22 @@ function storeItems (blockstore, items) {
     const fanoutLinks = []
 
     for (let i = 0; i < DEFAULT_FANOUT; i++) {
-      fanoutLinks.push(new DAGLink('', 1, EMPTY_KEY))
+      fanoutLinks.push({
```

Comment:
💭 Seems like having an utility function to create links would have been still useful.

Reply:
simple factories that match the old `DAGLink` and `DAGNode` constructors would probably be good enough to make this pain decrease a little eh?

Reply:
ok, since this one is simple I've drafted it up but would like feedback: https://github.com/ipld/js-dag-pb/pull/20

These 4 lines would become `fanoutLinks.push(createLink('', 1, EMPTY_KEY))`. Same below, and then the return at the end would be `return createNode(rootData, rootLinks)`.

Is this enough of an improvement to warrant API surface area increase?

Reply:
What I really meant is something like `link({ cid: EMPTY_KEY, size: 1, name: '' })` just as a sugar.